notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+# New GraphQL.org Site Architecture:
+
+## Index
+
+*Goal:* This is the landing page and is our opportunity to quickly capture attention and explain what GraphQL is and why you should care.
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
```

Comment:
I think this point would be stronger if it was a bit more subtle - some people are kind of scared off by the idea of "replacing" REST, but in many ways GraphQL can be quite similar. Perhaps it's more valuable to position GraphQL as an _upgrade_ to REST, rather than a replacement. This implies that people using REST should be excited to improve their situation by migrating to GraphQL incrementally, rather than saying it's a scary thing where they need to rewrite their whole aPI.